docs/data_structures/history_array.rst

```diff
@@ -85,32 +85,22 @@ The full list of these reserved fields is given below.
 * ``given`` [bool]: True if this ``gen_f`` output been given to a libEnsemble
```

Comment:
This file still needs to be adjusted, right?

Reply:
I have updated this file.